addOns/pscanrules/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- The Insecure JSF ViewState - P now includes example alert functionality for documentation generation purposes (Issue 6119).
+### Changed
```

Comment:
There's already a Changed heading this one can be removed.

Reply:
You can also drop the " - P" in the comment that was there in the table just so things could be located more easily.

Reply:
Now needs to be moved back to the `## Unreleased` section, and the " - P" still wasn't removed.

Reply:
Now you seem to have removed the v60 heading. The file should be in the following order:

Generic note about file
Unreleased heading
Changed heading
Your change note
v60 heading and notes

Reply:
See https://github.com/zaproxy/zap-extensions/pull/5688/files#r1743164814